plugins/linuxplugin/linuxplugin_init.go

```diff
@@ -81,7 +82,8 @@ func (plugin *Plugin) Init() error {
 		"linux_if_indexes", nil))
 
 	// Linux interface configurator
-	plugin.ifConfigurator = &LinuxInterfaceConfigurator{}
+	linuxLogger := plugin.Log.NewLogger("-iface-cfg")
```

Comment:
what about renaming to ifaceLogger?

Reply:
the full name shown in the log is  logger=linux-plugin-iface-cfg so it's clear that it is a logger. I can eventually rename it to linux-plugin-if-conf (similar to default plugins).